gradle/Gemfile

```diff
@@ -3,5 +3,6 @@
 source "https://rubygems.org"
 
 gem "dependabot-common", path: "../common"
+gem "dependabot-maven", path: "../maven"
```

Comment:
I think we may need to add this dependency to the `dependabot-gradle` gemspec.

E.g.
```ruby
spec.add_dependency "dependabot-maven", Dependabot::VERSION
```